pkg/apis/migration/v1alpha1/gvk.go

```diff
@@ -0,0 +1,29 @@
+package v1alpha1
+
+// UnsupportedNamespace - namespace, which is noticed
+// to contain resources unsupported by the migration
+type UnsupportedNamespace struct {
+	Name                 string   `json:"name"`
+	UnsupportedResources []string `json:"unsupportedResources"`
+}
+
+func (p *MigPlanStatus) SelectUnsupported() []string {
+	namespaces := []string{}
+	for _, ns := range p.UnsupportedNamespaces {
+		namespaces = append(namespaces, ns.Name)
+	}
+
+	return namespaces
+}
+
+func (p *MigPlanStatus) InitUnsupported() {
```

Comment:
:+1: for functional decomposition (generally) but don't think this function adds value.

Reply:
I consider this an initial run for GVK related tasks. This structure may contain additional fields, some specific info to `why` the resource is unsupported, and what needs to be done, which it does not have at the moment. May be the name change to `ResetUnsupported()` will be sufficient?